server/backend/api/app/routers/router_application.py

```diff
@@ -16,14 +16,14 @@
 def get_applications(
     response: Response,
     db: Session = Depends(database.get_db),
-    token_claims: dict = Depends(jwt_validation.authorize)
+    access_roles: dict = Depends(jwt_validation.get_access_roles)
```

Comment:
This change confuses me. Don't we want the get_applications call depending on an authorize check? But this seems to avoid any authorize check within jwt_validation. Oh wait, is that what the Depends(authorize) does within the get_access_roles...